packages/stream_chat_flutter_core/pubspec.yaml

```
@@ -11,7 +11,7 @@ environment:
 
 dependencies:
   collection: ^1.17.2
-  connectivity_plus: ">=4.0.2 <6.0.0"
+  connectivity_plus: ">=4.0.2 <7.0.0"
```

Shouldn't we require 6.0.0 now? It won't comple for versions before that because of the result being a list now.

```suggestion
  connectivity_plus: ">=6.0.1 <7.0.0"
```